### PR TITLE
Remove duplicate enums from Network specs

### PR DESCRIPTION
**Pull request by anton-evseev**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by dsgouda** [comment]
@shahabhijeet @NelsonDaniel PTAL

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteCircuit.json [hunk 1]
```diff
@@ -1751,18 +1751,9 @@
...
         },
         "circuitConnectionStatus": {
-          "type": "string",
-          "description": "Express Route Circuit Connection State. Possible values are: 'Connected' and 'Disconnected'.",
-          "enum": [
-            "Connected",
-            "Connecting",
-            "Disconnected"
-          ],
           "readOnly": true,
```
Comment (NelsonDaniel) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteCircuit.json]: The "readOnly" field should be moved out from here and placed in the definition. Every instance is set with "readOnly" = true.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteCircuit.json [hunk 2]
```diff
@@ -1827,18 +1818,9 @@
...
```
Comment (NelsonDaniel) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteCircuit.json]: The "readOnly" field should be moved out from here and placed in the definition. Every instance has "readOnly" = true.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteCircuit.json [hunk 3]
```diff
@@ -2258,6 +2230,33 @@
         }
       },
       "description": "Response for the ListExpressRouteServiceProvider API service call."
+    },
+    "CircuitConnectionStatus": {
+      "type": "string",
```
Comment (NelsonDaniel) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteCircuit.json]: The "readOnly" field should be placed here as every schema that references this definitions contains it.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteGateway.json [hunk 4]
```diff
@@ -595,19 +585,9 @@
             ],
             "properties": {
                 "provisioningState": {
-                    "type": "string",
                     "readOnly": true,
```
Comment (NelsonDaniel) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteGateway.json]: The definition already contains "readOnly": true. This should be removed from here.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteGateway.json [hunk 5]
```diff
@@ -536,19 +536,9 @@
                     }
                 },
                 "provisioningState": {
-                    "type": "string",
                     "readOnly": true,
```
Comment (NelsonDaniel) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteGateway.json]: The definition already contains "readOnly": true. This should be removed from here.

**Review by NelsonDaniel — CHANGES_REQUESTED** [review]
I left some comments @number213

**Comment by number213** [comment]
@NelsonDaniel addressed your comments, please take a look

**Review by NelsonDaniel — APPROVED** [review]
Looks good to me

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
@shahabhijeet Please approve if ready

**Comment by MikhailTryakhov** [comment]
@dsgouda @shahabhijeet  release is tomorrow, noon 
please help to merge in time

**Pull request merged**